src/App.tsx

```
@@ -1,26 +1,102 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useEffect } from 'react';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { getTodos, getUser } from './api';
+import { actions as todosActions } from './features/todos';
+import { actions as filterActions } from './features/filter';
+import { actions as currentTodoActions } from './features/currentTodo';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo.currentTodo);
+  const isUserLoading = useAppSelector(
+    state => state.currentTodo.isUserLoading,
+  );
+  const assignedUser = useAppSelector(state => state.currentTodo.assignedUser);
+  const searchValue = useAppSelector(state => state.filter.query);
+  const status = useAppSelector(state => state.filter.status);
+  const todos = useAppSelector(state => state.todos.todos);
+  const isTodosLoading = useAppSelector(state => state.todos.isTodosLoading);
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  useEffect(() => {
+    dispatch(todosActions.setIsTodosLoading(true));
+    getTodos()
+      .then(data => {
+        dispatch(todosActions.setTodos(data));
+      })
+      .catch()
```

Problem: `.catch()` is empty here which swallows errors. The app currently won't crash (finally resets loading) but you should at least log the error or dispatch an error state so failures are visible and debuggable. Consider: `.catch(error => { console.error(error); /* dispatch error flag */ })`.